frontend/src/components/TeamProject/TeamProject.tsx

```diff
@@ -0,0 +1,156 @@
+import React, { useEffect, useState } from 'react';
+import styles from './TeamProject.module.css'
+import { Button } from '@material-ui/core';
+import {
+  selectTeamProjects,
+  getProjectById,
+  saveProjectById,
+  deleteProjectById,
+  switchEditMode,
+  switchDeleteMode
+} from './TeamProjectSlice'
+import { useAppDispatch, useAppSelector } from '../../app/hooks';
+
+export interface TeamProjectProps {
+  _id: string, //id projektu wybranego w tabeli
+  changeViewFn: Function
+}
+
+interface deleteModalProps {
+  projectDeleteMode: boolean;
+  _id: string;
+}
+
+const TeamProject: React.FC<TeamProjectProps> = props => {
+  const { projectEditMode, projectDeleteMode, project } = useAppSelector(selectTeamProjects);
+  let selectedTeamProject = project;
+
+  const dispatch = useAppDispatch();
+  const [projectName, setProjectName] = useState(project.projectName);
+  const [projectUrl, setProjectUrl] = useState(project.projectUrl);
+  const [projectDescription, setProjectDescription] = useState(project.description);
+
+  // const token = window.localStorage....
+  const token = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJfaWQiOiI2MDRmYmFlMWEyZTM4ZDAwMTVlZTQxZWQiLCJ0eXBlIjozLCJpYXQiOjE2MTY3OTA3NDksImV4cCI6MTYxNjc5MTk0OX0.cYNseRM97U7IgwXKVQgRwBVd7SQWpeHJ4grgWUqsf6w'
```

Comment:
These lines are not necessary.

Reply:
done, I removed all the token mentions in the file as well